pkg/process/exec_conf.go

```diff
@@ -99,17 +99,20 @@ func saveConfig(flagset *pflag.FlagSet, outfile string, overrides map[string]int
 		if overriddenValue != nil {
 			value = fmt.Sprintf("%v", overriddenValue)
 		}
+		//print usage info
 		if f.Usage != "" {
 			fmt.Fprintf(w, "# %s\n", f.Usage)
 		}
-		fmt.Fprintf(w, "%s: ", k)
+		//print commented key (beginning of value assignement line)
+		fmt.Fprintf(w, "# %s: ", k)
```

Comment:
well, wait, k might not be the default here. i think we need an if check of some kind. when saveConfig is called, we're given overrides and so on. only the flagset knows what the true default is

Reply:
k?  k is the key - variable name... but yeah... this is all very Viper specific code that gets sketchier over time.

Reply:
oh shoot, you're right, i meant the line might not want to be commented out indiscriminately - it depends on if the value is actually different than the code default